app/models/miq_alert_status_state.rb

```diff
@@ -0,0 +1,7 @@
+class MiqAlertStatusState < ApplicationRecord
```

Comment:
> the whole naming of the alerts is confusing

If this is unavoidable on the domain-modelling layer (/cc @blomquisg) there is a chance to fix this on the API layer. Having confusingly-named endpoints is going to result in a poor UX, IMO. Since MiqAlerts are not exposed in the API, WDYT about mapping `MiqAlertStatus` -> `api/alerts` and `MiqAlertStatusState` -> `api/alert_statuses`?